cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/autocomplete_switch.spec.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  INDEX_WITH_TIME_1,
+  INDEX_WITH_TIME_2,
+  SECONDARY_ENGINE,
+  QueryLanguages,
+} from '../../../../../utils/constants';
+import {
+  getRandomizedWorkspaceName,
+  getRandomizedDatasourceName,
+  getDefaultQuery,
+} from '../../../../../utils/apps/query_enhancements/shared';
+import {
+  generateAutocompleteTestConfigurations,
+  LanguageConfigs,
+  getDatasetName,
+} from '../../../../../utils/apps/query_enhancements/autocomplete';
+
+const workspaceName = getRandomizedWorkspaceName();
+const datasourceName = getRandomizedDatasourceName();
```

Comment:
just fyi i am removing the function `getRandomizedDatasourceName()` in my other open PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/9319

but if your PR gets merged first, I will take care of it

Reply:
Yeah thanks for the remind